umich_api/api_utils.py

```diff
@@ -86,15 +117,14 @@ def api_call(self, api_call, client_scope, method="GET", payload=None, api_speci
         return this_scope.get("api_call")(api_call, self.get_access_token(this_scope.get("token_url"), client_scope), method, payload, api_specific_headers)
 
     def _api_call(self, api_call, access_token, method="GET", payload=None, api_specific_headers=None):
-        # type: (str, str, Dict[str, str], str, str) -> requests.Response
+        # type: (str, str, Dict[str, str], str, str) -> Response
         """ Internal method for making api calls 
         """
 
         if access_token is None:
             raise Exception("Must obtain an access token before making API Call")
 
         headers = {
-            "accept" : "application/json",
```

Comment:
Were you just removing this because it works without it? Or was this the default now?

Reply:
I removed this because it was common to both the OAuth token call and `api_call`, and we can set common headers with the session (see changes to the constructor, where I'm using `self.session.headers.update({ ... })`.